apps/cli/src/commands/rollups/start.ts

```diff
@@ -6,43 +6,18 @@ import pRetry from "p-retry";
 import path from "path";
 import { getServiceHealth } from "../../base.js";
 import { DEFAULT_SDK_IMAGE, DEFAULT_SDK_VERSION } from "../../config.js";
+import {
+    Service,
+    baseServices,
```

Comment:
I don't think we need to try to reuse code between rollups and coprocessor at this point yet.